src/components/Table/GridTable.test.tsx

```diff
@@ -2792,21 +2792,72 @@ describe("GridTable", () => {
 
       // Then the column is initially expanded (1 column + 1 expanded column)
       expect(row(r, 1).childNodes).toHaveLength(2);
+      expect(tableSnapshot(r)).toMatchInlineSnapshot(`
+        "
+        | Client name |
+        | First name  | Last name |
+        | ----------- | --------- |
+        | Brandon     | Dow       |
+        "
+      `);
 
       //  When clicking to collapse `columnA`
       click(r.expandableColumn);
 
       // Then the column is collapsed
       expect(row(r, 1).childNodes).toHaveLength(1);
+      expect(tableSnapshot(r)).toMatchInlineSnapshot(`
+              "
```

Comment:
Really minor question, but why do you think the `"` is indented here all the way under the `a` vs. under the `p` in `expect` like on line 2796? :shrug:

Reply:
ah I might have accidentally indented it when testing 🤔 
Testing a bit it looks like `toMatchInlineSnapshot` ignores the whitespace so the test passes either way. Generating from scratch does give the indentation we'd expect, will push that up.